templates/graphDebug.html

```diff
@@ -71,29 +69,20 @@
             runtimeOptions.defaultGraph = graph;
           };
           if (!queryProtocol) {
-              queryProtocol = 'webrtc';
+            queryProtocol = 'opener';
           }
 
           var rt = null;
-          if (queryProtocol == 'webrtc') {
-            var webrtcRuntime = require('noflo-runtime-webrtc');
-            if (queryAddress) {
-              // ID to use specified from outside, normally by Flowhub IDE
-              rt = webrtcRuntime(queryAddress, runtimeOptions, true);
```

Comment:
This removes WebRTC support. I guess that is OK, since it is not really used and is pretty flaky..